### ISSUE TITLE
Channel Check

### ISSUE THREAD

**Issue by enyonyelison** [Newcomer1989/TSN-Ranksystem#742]
how could the bot only work on one channel instead of all?

**Comment by Newcomer1989** [comment]
Cause it is a TS3 Server Query User.

**Comment by enyonyelison** [comment]
yes, even if it is a Query, would it be possible for the bot to work only on one channel, check only one channel instead of all?

**Comment by Newcomer1989** [comment]
Ok, I understood your first question a little bit different.

Could you more explain / describe, what exactly your are asking for? What function do you mean with 'work'?

**Comment by enyonyelison** [comment]
example, the bot it works for all rooms of the correct ts3.
I was wondering if it can work on only one channel.

**Comment by Newcomer1989** [comment]
Are you meaning the ranks itself?
If I understand you right, you are on the webinterface -> ranks -> exceptions -> channel execption
Here it is currently possible to except channel from the ranking, like a blacklist.
You wish a new feature to define here a whitelist, right?

**Comment by enyonyelison** [comment]
this, but instead of excluding the verification channels, he chooses only one to work, understand?

**Comment by enyonyelison** [comment]
Good Night, any update on the Whitelisting for Channel?
